src/std_extensions/arithmetic/int_types.rs

```diff
@@ -198,6 +198,12 @@ pub fn extension() -> Extension {
     extension
 }
 
+/// get an integer type variable, given the integer type definition
+pub(super) fn int_type_var(var_id: usize, int_type_def: &TypeDef) -> Result<Type, SignatureError> {
```

Comment:
You could make this find the `int_type_def`, right? (So we don't have to pass the values returned by the function around everywhere? I've commented elsewhere but it does seem a bit error-prone)